e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -104,24 +117,24 @@ var _ = Describe("App With Domain", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(len(svc.Routes)).To(Equal(1))
 
-			// Validate route has the expected HTTPS endpoint.
-			route := svc.Routes[0]
-			Expect(route.Environment).To(Equal(envName))
-			Expect(route.URL).To(Equal(fmt.Sprintf("https://%s", domainName)))
-
-			// Make sure the response is OK.
-			// Since the www app was added second, it should have app appended to the name.
-			var resp *http.Response
-			var fetchErr error
-			Eventually(func() (int, error) {
-				resp, fetchErr = http.Get(route.URL)
-				return resp.StatusCode, fetchErr
-			}, "60s", "1s").Should(Equal(200))
-			Eventually(func() (int, error) {
-				httpRoute := fmt.Sprintf("http://%s", domainName)
-				resp, fetchErr = http.Get(httpRoute)
-				return resp.StatusCode, fetchErr
-			}, "60s", "1s").Should(Equal(200))
+			for _, route := range svc.Routes {
+				// Validate route has the expected HTTPS endpoint.
+				Expect(route.URL).To(Equal(fmt.Sprintf("https://%s.%s", route.Environment, domainName)))
```

Comment:
nit: Should we have a map instead of expected endpoints so that we can always change the values in the manifest without breaking the pattern:
```go
wantedURLs := map[string]string {
  "test": "test.copilot-e2e-tests.ecs.aws.dev",
  "prod": "prod.copilot-e2e-tests.ecs.aws.dev",
}
```